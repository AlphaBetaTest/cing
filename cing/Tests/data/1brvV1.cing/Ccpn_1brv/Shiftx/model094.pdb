REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       4.999   0.426   5.151  1.00  0.00              
ATOM      2  CA  VAL A 171       4.386   0.995   3.927  1.00  0.00              
ATOM      3  HA  VAL A 171       3.884   1.954   4.151  1.00  0.00              
ATOM      4  CB  VAL A 171       5.567   1.370   2.969  1.00  0.00              
ATOM      5  HB  VAL A 171       5.074   1.699   2.054  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.413   2.570   3.459  1.00  0.00              
ATOM      7 HG11 VAL A 171       5.796   3.475   3.613  1.00  0.00              
ATOM      8 HG12 VAL A 171       6.923   2.354   4.417  1.00  0.00              
ATOM      9 HG13 VAL A 171       7.207   2.844   2.739  1.00  0.00              
ATOM     10  CG2 VAL A 171       6.528   0.238   2.536  1.00  0.00              
ATOM     11 HG21 VAL A 171       5.973  -0.641   2.162  1.00  0.00              
ATOM     12 HG22 VAL A 171       7.197   0.561   1.716  1.00  0.00              
ATOM     13 HG23 VAL A 171       7.169  -0.116   3.364  1.00  0.00              
ATOM     14  C   VAL A 171       3.230   0.081   3.357  1.00  0.00              
ATOM     15  O   VAL A 171       3.438  -0.609   2.355  1.00  0.00              
ATOM     16  N   PRO A 172       1.983   0.049   3.927  1.00  0.00              
ATOM     17  CD  PRO A 172       1.701   0.462   5.319  1.00  0.00              
ATOM     18  CA  PRO A 172       0.852  -0.751   3.377  1.00  0.00              
ATOM     19  HA  PRO A 172       1.208  -1.548   2.692  1.00  0.00              
ATOM     20  CB  PRO A 172       0.278  -1.415   4.654  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.820  -1.295   4.741  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.452  -2.507   4.645  1.00  0.00              
ATOM     23  CG  PRO A 172       0.993  -0.781   5.857  1.00  0.00              
ATOM     24  HG2 PRO A 172       0.301  -0.540   6.686  1.00  0.00              
ATOM     25  HG3 PRO A 172       1.741  -1.488   6.267  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.040   1.350   5.325  1.00  0.00              
ATOM     27  HD3 PRO A 172       2.576   0.716   5.942  1.00  0.00              
ATOM     28  C   PRO A 172      -0.206   0.131   2.635  1.00  0.00              
ATOM     29  O   PRO A 172      -0.814   1.021   3.242  1.00  0.00              
ATOM     30  N   CYS A 173      -0.494  -0.146   1.350  1.00  0.00              
ATOM     31  H   CYS A 173       0.125  -0.809   0.865  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.532   0.590   0.569  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.287   1.656   0.567  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.370   0.197  -0.896  1.00  0.00              
ATOM     35  HB2 CYS A 173      -0.715  -0.661  -1.052  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.344  -0.031  -1.307  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.552   1.490  -1.820  1.00  0.00              
ATOM     38  C   CYS A 173      -2.993   0.539   1.137  1.00  0.00              
ATOM     39  O   CYS A 173      -3.696   1.552   1.090  1.00  0.00              
ATOM     40  N   SER A 174      -3.413  -0.595   1.727  1.00  0.00              
ATOM     41  H   SER A 174      -2.697  -1.327   1.771  1.00  0.00              
ATOM     42  CA  SER A 174      -4.668  -0.676   2.541  1.00  0.00              
ATOM     43  HA  SER A 174      -5.514  -0.370   1.910  1.00  0.00              
ATOM     44  CB  SER A 174      -4.863  -2.154   2.942  1.00  0.00              
ATOM     45  HB2 SER A 174      -5.778  -2.247   3.562  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.048  -2.758   2.038  1.00  0.00              
ATOM     47  OG  SER A 174      -3.746  -2.682   3.670  1.00  0.00              
ATOM     48  HG  SER A 174      -3.625  -3.603   3.384  1.00  0.00              
ATOM     49  C   SER A 174      -4.787   0.236   3.815  1.00  0.00              
ATOM     50  O   SER A 174      -5.847   0.290   4.444  1.00  0.00              
ATOM     51  N   THR A 175      -3.712   0.942   4.181  1.00  0.00              
ATOM     52  H   THR A 175      -2.893   0.739   3.596  1.00  0.00              
ATOM     53  CA  THR A 175      -3.672   1.913   5.311  1.00  0.00              
ATOM     54  HA  THR A 175      -4.685   1.959   5.753  1.00  0.00              
ATOM     55  CB  THR A 175      -2.726   1.393   6.435  1.00  0.00              
ATOM     56  HB  THR A 175      -2.824   2.076   7.303  1.00  0.00              
ATOM     57  OG1 THR A 175      -1.369   1.415   6.013  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.358   1.133   5.089  1.00  0.00              
ATOM     59  CG2 THR A 175      -3.038  -0.034   6.928  1.00  0.00              
ATOM     60 HG21 THR A 175      -4.094  -0.130   7.243  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.885  -0.786   6.129  1.00  0.00              
ATOM     62 HG23 THR A 175      -2.397  -0.324   7.777  1.00  0.00              
ATOM     63  C   THR A 175      -3.334   3.391   4.872  1.00  0.00              
ATOM     64  O   THR A 175      -3.618   4.319   5.631  1.00  0.00              
ATOM     65  N   CYS A 176      -2.736   3.636   3.678  1.00  0.00              
ATOM     66  H   CYS A 176      -2.814   2.865   3.010  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.504   5.000   3.126  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.984   5.615   3.886  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.532   4.748   1.949  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.903   4.057   1.200  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.338   5.625   1.306  1.00  0.00              
ATOM     72  SG  CYS A 176       0.049   4.172   2.589  1.00  0.00              
ATOM     73  C   CYS A 176      -3.818   5.781   2.762  1.00  0.00              
ATOM     74  O   CYS A 176      -3.865   7.006   2.881  1.00  0.00              
ATOM     75  N   GLU A 177      -4.863   5.064   2.305  1.00  0.00              
ATOM     76  H   GLU A 177      -4.623   4.101   2.058  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.246   5.569   2.098  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.677   4.795   1.446  1.00  0.00              
ATOM     79  CB  GLU A 177      -7.004   5.522   3.473  1.00  0.00              
ATOM     80  HB2 GLU A 177      -6.298   5.520   4.328  1.00  0.00              
ATOM     81  HB3 GLU A 177      -7.565   6.462   3.633  1.00  0.00              
ATOM     82  CG  GLU A 177      -8.012   4.354   3.637  1.00  0.00              
ATOM     83  HG2 GLU A 177      -8.570   4.479   4.584  1.00  0.00              
ATOM     84  HG3 GLU A 177      -8.780   4.401   2.845  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.395   2.958   3.615  1.00  0.00              
ATOM     86  OE1 GLU A 177      -7.050   2.384   2.586  1.00  0.00              
ATOM     87  OE2 GLU A 177      -7.263   2.427   4.858  1.00  0.00              
ATOM     88  C   GLU A 177      -6.468   6.858   1.256  1.00  0.00              
ATOM     89  O   GLU A 177      -7.227   7.766   1.603  1.00  0.00              
ATOM     90  N   GLY A 178      -5.826   6.870   0.085  1.00  0.00              
ATOM     91  H   GLY A 178      -4.959   6.326   0.150  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.877   8.034  -0.821  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.877   7.632  -1.844  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.834   8.556  -0.713  1.00  0.00              
ATOM     95  C   GLY A 178      -4.737   9.072  -0.722  1.00  0.00              
ATOM     96  O   GLY A 178      -4.767  10.065  -1.450  1.00  0.00              
ATOM     97  N   ASN A 179      -3.722   8.848   0.134  1.00  0.00              
ATOM     98  H   ASN A 179      -3.660   7.877   0.455  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.537   9.715   0.211  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.785  10.740  -0.121  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.904   9.622   1.613  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.656   9.471   2.412  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.274   8.707   1.580  1.00  0.00              
ATOM    104  CG  ASN A 179      -0.985  10.788   1.943  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.174  10.830   1.550  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.479  11.775   2.637  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.460  11.689   2.915  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.818  12.532   2.820  1.00  0.00              
ATOM    109  C   ASN A 179      -1.554   9.158  -0.803  1.00  0.00              
ATOM    110  O   ASN A 179      -0.912   8.120  -0.605  1.00  0.00              
ATOM    111  N   LEU A 180      -1.431   9.874  -1.896  1.00  0.00              
ATOM    112  H   LEU A 180      -2.058  10.677  -1.986  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.640   9.374  -3.013  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.065   8.362  -3.077  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.045  10.296  -4.172  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.185  11.335  -3.789  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.203  10.319  -4.833  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.245   9.873  -5.028  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.032   9.687  -4.299  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.687  11.012  -5.960  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.954  11.925  -5.396  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.894  11.292  -6.680  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.581  10.736  -6.551  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.998   8.597  -5.852  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.722   7.734  -5.220  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.898   8.295  -6.420  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.181   8.730  -6.587  1.00  0.00              
ATOM    128  C   LEU A 180       0.893   9.154  -2.810  1.00  0.00              
ATOM    129  O   LEU A 180       1.497   8.320  -3.476  1.00  0.00              
ATOM    130  N   ALA A 181       1.474   9.876  -1.854  1.00  0.00              
ATOM    131  H   ALA A 181       0.762  10.449  -1.403  1.00  0.00              
ATOM    132  CA  ALA A 181       2.834   9.617  -1.337  1.00  0.00              
ATOM    133  HA  ALA A 181       3.504   9.516  -2.204  1.00  0.00              
ATOM    134  CB  ALA A 181       3.253  10.864  -0.551  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.229  11.769  -1.184  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.596  11.038   0.322  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.284  10.750  -0.171  1.00  0.00              
ATOM    138  C   ALA A 181       2.916   8.287  -0.503  1.00  0.00              
ATOM    139  O   ALA A 181       3.733   7.430  -0.826  1.00  0.00              
ATOM    140  N   CYS A 182       2.047   8.085   0.515  1.00  0.00              
ATOM    141  H   CYS A 182       1.239   8.718   0.463  1.00  0.00              
ATOM    142  CA  CYS A 182       1.912   6.789   1.258  1.00  0.00              
ATOM    143  HA  CYS A 182       2.777   6.539   1.831  1.00  0.00              
ATOM    144  CB  CYS A 182       0.873   7.061   2.339  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.036   8.029   2.824  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.112   7.137   1.885  1.00  0.00              
ATOM    147  SG  CYS A 182       0.813   5.734   3.554  1.00  0.00              
ATOM    148  C   CYS A 182       1.673   5.537   0.361  1.00  0.00              
ATOM    149  O   CYS A 182       2.354   4.513   0.458  1.00  0.00              
ATOM    150  N   LEU A 183       0.708   5.690  -0.533  1.00  0.00              
ATOM    151  H   LEU A 183       0.204   6.584  -0.457  1.00  0.00              
ATOM    152  CA  LEU A 183       0.403   4.751  -1.620  1.00  0.00              
ATOM    153  HA  LEU A 183       0.123   3.780  -1.180  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.812   5.441  -2.271  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.533   6.467  -2.556  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.984   4.989  -3.203  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.187   5.415  -1.604  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.019   5.943  -0.651  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.189   6.171  -2.490  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.206   5.775  -3.525  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.218   6.088  -2.107  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.950   7.246  -2.564  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.722   3.983  -1.413  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.036   3.358  -0.822  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.693   3.961  -0.887  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.855   3.457  -2.378  1.00  0.00              
ATOM    167  C   LEU A 183       1.540   4.494  -2.678  1.00  0.00              
ATOM    168  O   LEU A 183       1.614   3.402  -3.238  1.00  0.00              
ATOM    169  N   SER A 184       2.382   5.497  -2.977  1.00  0.00              
ATOM    170  H   SER A 184       2.180   6.331  -2.406  1.00  0.00              
ATOM    171  CA  SER A 184       3.589   5.321  -3.849  1.00  0.00              
ATOM    172  HA  SER A 184       3.291   4.732  -4.739  1.00  0.00              
ATOM    173  CB  SER A 184       4.066   6.695  -4.366  1.00  0.00              
ATOM    174  HB2 SER A 184       3.238   7.229  -4.869  1.00  0.00              
ATOM    175  HB3 SER A 184       4.379   7.344  -3.524  1.00  0.00              
ATOM    176  OG  SER A 184       5.144   6.548  -5.296  1.00  0.00              
ATOM    177  HG  SER A 184       5.683   5.805  -4.994  1.00  0.00              
ATOM    178  C   SER A 184       4.771   4.524  -3.190  1.00  0.00              
ATOM    179  O   SER A 184       5.537   3.876  -3.908  1.00  0.00              
ATOM    180  N   LEU A 185       4.962   4.617  -1.863  1.00  0.00              
ATOM    181  H   LEU A 185       4.266   5.211  -1.388  1.00  0.00              
ATOM    182  CA  LEU A 185       5.934   3.758  -1.112  1.00  0.00              
ATOM    183  HA  LEU A 185       6.869   3.645  -1.664  1.00  0.00              
ATOM    184  CB  LEU A 185       6.221   4.390   0.286  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.346   4.187   0.935  1.00  0.00              
ATOM    186  HB3 LEU A 185       7.052   3.822   0.743  1.00  0.00              
ATOM    187  CG  LEU A 185       6.539   5.900   0.407  1.00  0.00              
ATOM    188  HG  LEU A 185       5.668   6.450   0.002  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.690   6.301   1.882  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.776   6.072   2.461  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.532   5.777   2.374  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.868   7.387   1.993  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.799   6.305  -0.374  1.00  0.00              
ATOM    194 HD21 LEU A 185       8.698   5.764  -0.026  1.00  0.00              
ATOM    195 HD22 LEU A 185       7.687   6.097  -1.454  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.006   7.388  -0.282  1.00  0.00              
ATOM    197  C   LEU A 185       5.521   2.266  -0.921  1.00  0.00              
ATOM    198  O   LEU A 185       6.376   1.385  -0.780  1.00  0.00              
ATOM    199  N   CYS A 186       4.213   2.012  -0.883  1.00  0.00              
ATOM    200  H   CYS A 186       3.685   2.826  -0.565  1.00  0.00              
ATOM    201  CA  CYS A 186       3.664   0.723  -0.536  1.00  0.00              
ATOM    202  HA  CYS A 186       4.166   0.546   0.422  1.00  0.00              
ATOM    203  CB  CYS A 186       2.157   1.052  -0.391  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.735   0.292   0.267  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.871   2.000   0.086  1.00  0.00              
ATOM    206  SG  CYS A 186       1.345   0.935  -2.002  1.00  0.00              
ATOM    207  C   CYS A 186       3.784  -0.569  -1.405  1.00  0.00              
ATOM    208  O   CYS A 186       4.011  -0.602  -2.616  1.00  0.00              
ATOM    209  N   HIS A 187       3.455  -1.639  -0.682  1.00  0.00              
ATOM    210  H   HIS A 187       3.708  -1.502   0.303  1.00  0.00              
ATOM    211  CA  HIS A 187       3.220  -2.994  -1.223  1.00  0.00              
ATOM    212  HA  HIS A 187       3.526  -3.054  -2.278  1.00  0.00              
ATOM    213  CB  HIS A 187       4.097  -4.003  -0.433  1.00  0.00              
ATOM    214  HB2 HIS A 187       3.739  -4.041   0.588  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.925  -5.011  -0.821  1.00  0.00              
ATOM    216  CG  HIS A 187       5.599  -3.729  -0.383  1.00  0.00              
ATOM    217  ND1 HIS A 187       6.137  -2.924   0.604  1.00  0.00              
ATOM    218  CD2 HIS A 187       6.351  -3.648  -1.553  1.00  0.00              
ATOM    219  CE1 HIS A 187       7.132  -2.332  -0.120  1.00  0.00              
ATOM    220  NE2 HIS A 187       7.386  -2.741  -1.403  1.00  0.00              
ATOM    221  HD2 HIS A 187       5.904  -3.949  -2.483  1.00  0.00              
ATOM    222  HE1 HIS A 187       7.457  -1.357   0.212  1.00  0.00              
ATOM    223  C   HIS A 187       1.674  -3.291  -1.141  1.00  0.00              
ATOM    224  O   HIS A 187       0.923  -2.687  -0.358  1.00  0.00              
ATOM    225  N   ILE A 188       1.177  -4.306  -1.869  1.00  0.00              
ATOM    226  H   ILE A 188       1.737  -4.640  -2.652  1.00  0.00              
ATOM    227  CA  ILE A 188      -0.279  -4.611  -1.943  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.858  -3.673  -1.779  1.00  0.00              
ATOM    229  CB  ILE A 188      -0.534  -5.026  -3.443  1.00  0.00              
ATOM    230  HB  ILE A 188      -0.123  -4.216  -4.081  1.00  0.00              
ATOM    231  CG2 ILE A 188       0.136  -6.354  -3.908  1.00  0.00              
ATOM    232 HG21 ILE A 188       1.223  -6.375  -3.708  1.00  0.00              
ATOM    233 HG22 ILE A 188      -0.297  -7.240  -3.407  1.00  0.00              
ATOM    234 HG23 ILE A 188       0.021  -6.518  -4.995  1.00  0.00              
ATOM    235  CG1 ILE A 188      -2.048  -5.059  -3.732  1.00  0.00              
ATOM    236 HG12 ILE A 188      -2.476  -5.935  -3.211  1.00  0.00              
ATOM    237 HG13 ILE A 188      -2.503  -4.169  -3.260  1.00  0.00              
ATOM    238  CD1 ILE A 188      -2.450  -5.057  -5.219  1.00  0.00              
ATOM    239 HD11 ILE A 188      -2.051  -4.175  -5.754  1.00  0.00              
ATOM    240 HD12 ILE A 188      -2.088  -5.956  -5.751  1.00  0.00              
ATOM    241 HD13 ILE A 188      -3.550  -5.039  -5.339  1.00  0.00              
ATOM    242  C   ILE A 188      -0.711  -5.620  -0.811  1.00  0.00              
ATOM    243  O   ILE A 188      -0.976  -6.806  -1.015  1.00  0.00              
ATOM    244  N   GLU A 189      -0.797  -5.077   0.416  1.00  0.00              
ATOM    245  H   GLU A 189      -0.295  -4.188   0.471  1.00  0.00              
ATOM    246  CA  GLU A 189      -1.062  -5.850   1.651  1.00  0.00              
ATOM    247  HA  GLU A 189      -0.702  -6.884   1.516  1.00  0.00              
ATOM    248  CB  GLU A 189      -0.207  -5.196   2.777  1.00  0.00              
ATOM    249  HB2 GLU A 189      -0.444  -4.114   2.837  1.00  0.00              
ATOM    250  HB3 GLU A 189      -0.525  -5.597   3.760  1.00  0.00              
ATOM    251  CG  GLU A 189       1.331  -5.404   2.654  1.00  0.00              
ATOM    252  HG2 GLU A 189       1.590  -6.459   2.855  1.00  0.00              
ATOM    253  HG3 GLU A 189       1.669  -5.220   1.617  1.00  0.00              
ATOM    254  CD  GLU A 189       2.160  -4.527   3.590  1.00  0.00              
ATOM    255  OE1 GLU A 189       2.224  -4.689   4.803  1.00  0.00              
ATOM    256  OE2 GLU A 189       2.809  -3.534   2.923  1.00  0.00              
ATOM    257  C   GLU A 189      -2.577  -5.915   1.999  1.00  0.00              
TER     258      GLU A 189 
END   
