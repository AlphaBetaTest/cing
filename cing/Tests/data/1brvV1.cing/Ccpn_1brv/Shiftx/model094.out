# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0598
  171   V  CA      60.6387
  171   V  CB      32.4254
  171   V  C      172.1488
  171   V  H        7.8976
  171   V  HA       4.0649
  171   V  HB       2.1105
  171   V  HG11     0.9429
  171   V  HG12     0.9717
  171   V  HG13     0.2779
  171   V  HG21     1.0103
  171   V  HG22     1.0860
  171   V  HG23     0.9858
  172   P  CA      59.5820
  172   P  CB      33.4614
  172   P  C      176.6511
  172   P  HA       5.0931
  172   P  HB2      2.3576
  172   P  HB3      2.3223
  172   P  HD2      3.6069
  172   P  HD3      3.7358
  172   P  HG2      1.9860
  172   P  HG3      2.1332
  173   C  N      122.2917
  173   C  CA      59.8035
  173   C  CB      40.8643
  173   C  C      177.1036
  173   C  H        8.4778
  173   C  HA       4.3181
  173   C  HB2      3.0954
  173   C  HB3      3.1692
  174   S  N      113.7079
  174   S  CA      60.2720
  174   S  CB      62.5573
  174   S  C      175.0599
  174   S  H        8.1201
  174   S  HA       4.1137
  174   S  HB2      4.0176
  174   S  HB3      3.9941
  175   T  N      111.5850
  175   T  CA      62.2779
  175   T  CB      70.2895
  175   T  C      175.1391
  175   T  H        7.6797
  175   T  HA       4.6249
  175   T  HB       4.2406
  175   T  HG21     1.2112
  175   T  HG22     1.2118
  175   T  HG23     1.2121
  176   C  N      119.0129
  176   C  CA      57.7671
  176   C  CB      40.6016
  176   C  C      173.4567
  176   C  H        7.8913
  176   C  HA       4.3453
  176   C  HB2      3.1108
  176   C  HB3      3.2084
  177   E  N      118.0651
  177   E  CA      57.4715
  177   E  CB      26.4845
  177   E  C      175.2715
  177   E  H        9.4697
  177   E  HA       3.7137
  177   E  HB2      2.0542
  177   E  HB3      1.9850
  177   E  HG2      2.2226
  177   E  HG3      2.3414
  178   G  N      104.1867
  178   G  CA      45.3681
  178   G  C      173.5077
  178   G  H        8.3992
  178   G  HA    -666.0000
  178   G  HA2      3.9577
  178   G  HA3      3.9570
  179   N  N      119.1523
  179   N  CA      52.3532
  179   N  CB      39.3447
  179   N  C      175.3471
  179   N  H        7.8746
  179   N  HA       4.6002
  179   N  HB2      2.8017
  179   N  HB3      2.7282
  179   N  HD21     7.1705
  179   N  HD22     6.7945
  180   L  N      129.1481
  180   L  CA      57.2718
  180   L  CB      41.7265
  180   L  C      179.4306
  180   L  H        8.4563
  180   L  HA       3.9648
  180   L  HB2      1.7227
  180   L  HB3      1.7005
  180   L  HD11     0.9175
  180   L  HD12     0.9173
  180   L  HD13     0.9174
  180   L  HD21     0.2257
  180   L  HD22     1.9649
  180   L  HD23     0.8657
  180   L  HG       1.7220
  181   A  N      120.7176
  181   A  CA      55.2213
  181   A  CB      18.5374
  181   A  C      179.6506
  181   A  H        7.9044
  181   A  HA       3.9008
  181   A  HB1      1.3870
  181   A  HB2      1.3639
  181   A  HB3      1.3325
  182   C  N      116.4269
  182   C  CA      60.1617
  182   C  CB      43.4589
  182   C  C      175.9552
  182   C  H        7.9635
  182   C  HA       4.0847
  182   C  HB2      3.0096
  182   C  HB3      3.2310
  183   L  N      119.0111
  183   L  CA      57.8214
  183   L  CB      41.1985
  183   L  C      179.2213
  183   L  H        7.9307
  183   L  HA       3.9334
  183   L  HB2      1.8614
  183   L  HB3      1.7094
  183   L  HD11     0.9140
  183   L  HD12     0.9855
  183   L  HD13     0.9007
  183   L  HD21     0.3427
  183   L  HD22     2.3833
  183   L  HD23     0.8615
  183   L  HG       1.6725
  184   S  N      112.3887
  184   S  CA      61.1207
  184   S  CB      62.3774
  184   S  C      175.8719
  184   S  H        7.8710
  184   S  HA       4.2333
  184   S  HB2      4.0053
  184   S  HB3      4.0069
  185   L  N      120.9739
  185   L  CA      56.4203
  185   L  CB      41.5526
  185   L  C      175.1209
  185   L  H        7.6451
  185   L  HA       4.1698
  185   L  HB2      1.6352
  185   L  HB3      1.7052
  185   L  HD11     0.9175
  185   L  HD12     0.9209
  185   L  HD13     0.9187
  185   L  HD21     0.3279
  185   L  HD22     2.5805
  185   L  HD23     0.8682
  185   L  HG       1.7978
  186   C  N      110.7074
  186   C  CA      55.1066
  186   C  CB      37.8587
  186   C  C      172.2534
  186   C  H        7.0351
  186   C  HA       5.1801
  186   C  HB2      3.2452
  186   C  HB3      3.2612
  187   H  N      116.6159
  187   H  CA      54.1516
  187   H  CB      30.9747
  187   H  C      174.5298
  187   H  H        8.1032
  187   H  HA       4.8127
  187   H  HB2      3.4110
  187   H  HB3      3.2466
  187   H  HD2      5.8008
  187   H  HE1      6.7217
  188   I  N      124.9095
  188   I  CA      60.2156
  188   I  CB      36.0008
  188   I  C      176.1081
  188   I  H        7.9692
  188   I  HA       4.1440
  188   I  HB       1.8725
  188   I  HD11     0.8599
  188   I  HD12     0.8628
  188   I  HD13     0.8609
  188   I  HG12     1.2771
  188   I  HG13     0.7795
  188   I  HG21     0.9279
  188   I  HG22     0.9204
  188   I  HG23     0.9182
 *189   E  N      120.7907
 *189   E  CA      55.6781
 *189   E  CB      30.2999
 *189   E  C      176.5875
 *189   E  H        8.0120
 *189   E  HA       4.1747
 *189   E  HB2      1.9213
 *189   E  HB3      1.8485
 *189   E  HG2      2.2332
 *189   E  HG3      2.4083
